src/GovUk.Education.ExploreEducationStatistics.Common/Cache/MemoryCacheAttribute.cs

```diff
@@ -65,11 +65,14 @@ public static void ClearServices()
 
         public static void SetOverrideConfiguration(IConfigurationSection? configurationSection)
         {
-            OverrideDurationInSeconds = configurationSection?.GetValue<int?>("DurationInSeconds");
-
+            var durationInSeconds = configurationSection?.GetValue<int?>("DurationInSeconds");
+            
+            OverrideDurationInSeconds = durationInSeconds != null && durationInSeconds != -1 
+                ? durationInSeconds.Value : null;
+            
             var overrideExpirySchedule = configurationSection?.GetValue<string?>("ExpirySchedule");
 
-            OverrideExpirySchedule = overrideExpirySchedule != null 
+            OverrideExpirySchedule = overrideExpirySchedule != null && !"".Equals(overrideExpirySchedule)
```

Comment:
Not an issue but you could use `string.IsNullOrEmpty` here

`OverrideExpirySchedule = string.IsNullOrEmpty(overrideExpirySchedule) ? null : CrontabSchedule.Parse(overrideExpirySchedule)`

or

`OverrideExpirySchedule = overrideExpirySchedule.IsNullOrEmpty() ? null : CrontabSchedule.Parse(overrideExpirySchedule)`